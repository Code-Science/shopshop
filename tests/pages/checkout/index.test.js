import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import React from 'react';
import CheckoutPage from '../../../pages/checkout/index';

configure({ adapter: new Adapter() });

describe('CheckoutPage Component', () => {
  const wrapper = shallow(<CheckoutPage />);

  it('expect to render CheckoutPage component', () => {
    expect(wrapper.length).toEqual(1);
  });
  it('snapshot test', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
