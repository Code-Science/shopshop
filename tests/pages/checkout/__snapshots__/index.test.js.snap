// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckoutPage Component snapshot test 1`] = `
<div
  className="checkout"
>
  checkout
  <div
    className="stripeForm"
  >
    <h1
      className="orderSummaryHeading"
    >
      CHECKOUT
    </h1>
    <Elements
      stripe={Promise {}}
    >
      <CheckoutForm />
    </Elements>
    <OrderSummary />
  </div>
</div>
`;
